### ISSUE TITLE
Lifestyle, Income, Downtime and Time passing

### ISSUE THREAD

**Issue by crimsonknave** [our-stories/west-marches#46]
I'd really like to have downtime activities that we can have our characters do. Xanathar's has a good section on downtime activities: https://www.dndbeyond.com/sources/xgte/downtime-revisited#DowntimeActivities. I feel that they'll add some good flavor to the game, but we need to figure out what cadence we want for this sort of thing.

For example, Quay has a job in the ring and also performs many nights. Clearly, it's overkill to roll for each of these every day. He also lives a moderately extravagant lifestyle (at least for what Waldenstone can provide). It's unclear to me if he's net positive or negative. 

I'm used to downtime and lifestyles mattering between sessions as that's typically when time passes, however, time passing is a funny thing with our setup where a character can be in the middle of a session and go do another. We could base it on time passing in the real world. Have everyone have a downtime activity every month and declare how they're living and what they're doing for money. Then in that issue (or wherever) we can roll for downtime, and determine each character's money gain or loss based on lifestyle, jobs and downtime (Amberscale might spend $$$ trying to locate a rare scroll for example).

The monthly thing is just off the top of my head, would love to hear other ideas. I think we should also try and figure out how we want time to pass in general. It would be nice to have season come and go and we'll need some mechanics to determine when a scouting party or trade caravan comes back if we're waiting on them. It could be really jarring if we don't have a general consensus of how time is passing.

**Comment by thyeggman** [comment]
My long-running group uses the downtime rules you linked, they usually happen at a cadence of once every 2-3 sessions, say after traveling or waiting for the next ship to set sail.  I find the rules pretty good, they give a good amount of room for interesting role playing or narration instead of just rolling dice.  I don't think all the downtime has to be role-played, though, just whatever would be interesting.

**Comment by zmoazeni** [comment]
Thank you both for starting and adding to this conversation! I need to re-read Xanathar's rules. I remember going across it a while back, and I just haven't come back to it.

I'll link another living community that I know of (shadowrun though): [/r/runnerhub](https://www.reddit.com/r/RunnerHub/) and their [lifestyle rules](https://runnerhub.neosynth.net/index.php?n=Rules.Lifestyles).

We could set something up automatically for users who want to participate. Like an automatic issue/discussion that messages a group of users to remind them. I would want us to make this opt-in, and not mandatory.

So instead of lifestyles being required, you all could opt in with your characters for some sort of bonus. Like 10%-15% XP boost? I think there should be an incentive for that.

That said, I don't really want to police it. I would like it to stay on the honor system. Cheaters gonna cheat, and I don't want to have to audit character's balance sheets.

**Comment by djigneo** [comment]
Summary of a conversation I had with @crimsonknave :

- We need to come up with some way to have consistency with time passing. Perhaps a tie-out to real life time could be used. For example, every month of real-life time could represent 2 months of game time. That allows some shuffling to happen of sessions that have happened within a single month to have them make sense for the character, as well as an alottment of downtime to happen in addition to adventuring. (It would also allow for two month-long periods of downtime for players who were unable to attend a session.)
- For a period of downtime, players that do not specify any downtime activities will neither spend nor make gold for characters. It is assumed they are working a job that covers their lifestyle expenses but no more.
- For a period of downtime that players specify downtime activities, they are resolved via Xanathar's Guide to Everything at end of real-life period of time.
- Generally, downtime is important for games like this because it helps the players to drive the goals and sessions.

General economics for player characters as I understand them:

- A PC is an adventurer, which means they to some degree make ends meet by going on adventures rather than working a "real" job. An adventure tends to be more lucrative than a 9-5, but also more dangerous. Generally an adventurer would go on an excursion, then have a period of downtime.

- A decent baseline is to assume 1 adventure per month, with a Modest lifestyle. That would mean an adventurer would need about 30 gp income per month, so would expect to get around 30 gp profit from an adventure once per month to break even.

- To do some of the cool downtime activities, an adventurer would not be able to do their side-profession, so would effectively have to pay out of pocket to do their downtime activity. For example, Blaine "Scoop" Warrell wants to do some carousing in his downtime to uncover some rumors about the long-lost fortune of Brownbeard the Pirate King. Perhaps his contacts are high-class, so there could be a ruling that he has to pay Comfortable or Wealthy lifestyle expenses for part of the downtime, but would have a chance at generating a lead or perhaps some contacts from the carousing.
